2021-12-29 15:59:05,965 INFO    MainThread:867680 [wandb_setup.py:_flush():71] setting env: {}
2021-12-29 15:59:05,965 INFO    MainThread:867680 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-29 15:59:05,965 INFO    MainThread:867680 [wandb_init.py:_log_setup():371] Logging user logs to YOUR_ROOT/Code/lenas/RTDosePrediction/Src/Vnet/wandb/run-20211229_155905-25t8qgzk/logs/debug.log
2021-12-29 15:59:05,966 INFO    MainThread:867680 [wandb_init.py:_log_setup():372] Logging internal logs to YOUR_ROOT/Code/lenas/RTDosePrediction/Src/Vnet/wandb/run-20211229_155905-25t8qgzk/logs/debug-internal.log
2021-12-29 15:59:05,966 INFO    MainThread:867680 [wandb_init.py:init():404] calling init triggers
2021-12-29 15:59:05,966 INFO    MainThread:867680 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-29 15:59:05,966 INFO    MainThread:867680 [wandb_init.py:init():460] starting backend
2021-12-29 15:59:05,966 INFO    MainThread:867680 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-29 15:59:05,981 INFO    MainThread:867680 [backend.py:ensure_launched():216] starting backend process...
2021-12-29 15:59:05,993 INFO    MainThread:867680 [backend.py:ensure_launched():221] started backend process with pid: 867708
2021-12-29 15:59:05,995 INFO    MainThread:867680 [wandb_init.py:init():469] backend started and connected
2021-12-29 15:59:05,997 INFO    MainThread:867680 [wandb_init.py:init():533] updated telemetry
2021-12-29 15:59:06,021 INFO    MainThread:867680 [wandb_init.py:init():563] communicating current version
2021-12-29 15:59:06,738 INFO    MainThread:867680 [wandb_init.py:init():568] got version response 
2021-12-29 15:59:06,739 INFO    MainThread:867680 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2021-12-29 15:59:07,022 INFO    MainThread:867680 [wandb_init.py:init():606] starting run threads in backend
2021-12-29 15:59:10,503 INFO    MainThread:867680 [wandb_run.py:_console_start():1810] atexit reg
2021-12-29 15:59:10,504 INFO    MainThread:867680 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2021-12-29 15:59:10,504 INFO    MainThread:867680 [wandb_run.py:_redirect():1689] Redirecting console.
2021-12-29 15:59:10,506 INFO    MainThread:867680 [wandb_run.py:_redirect():1745] Redirects installed.
2021-12-29 15:59:10,506 INFO    MainThread:867680 [wandb_init.py:init():633] run started, returning control to user process
2021-12-29 15:59:24,649 INFO    MainThread:867680 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2021-12-29 15:59:24,652 INFO    MainThread:867680 [wandb_run.py:_restore():1752] restore
2021-12-29 15:59:27,592 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 853
}

2021-12-29 15:59:27,801 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 853
}

2021-12-29 15:59:28,065 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 853
}

2021-12-29 15:59:29,078 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 21319
}

2021-12-29 15:59:29,181 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 21319
}

2021-12-29 15:59:29,285 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 853
  total_bytes: 21319
}

2021-12-29 15:59:29,387 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:29,491 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:29,593 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:29,702 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:29,805 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:30,293 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}

2021-12-29 15:59:30,844 INFO    MainThread:867680 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21319
  total_bytes: 21319
}
local_info {
}

2021-12-29 15:59:32,032 INFO    MainThread:867680 [wandb_run.py:_append_files():2180] logging synced files
